docker-compose/common/keycloak-default-realm.json

```diff
@@ -0,0 +1,1681 @@
+        "composites" : {
+          "client" : {
+            "realm-management" : [ "view-clients", "manage-authorization", "view-realm", "manage-events", "create-client", "query-realms", "manage-identity-providers", "impersonation", "manage-realm", "query-users", "query-groups", "view-authorization", "query-clients", "manage-users", "view-events", "manage-clients", "view-identity-providers", "view-users" ]
+          }
+        },
+        "clientRole" : true,
+        "containerId" : "9bac98c6-31db-4742-b3ae-1247cfca5d9f",
+        "attributes" : { }
+      }, {
+        "id" : "f7307138-f50a-4033-8da0-5140e209aaca",
+        "name" : "view-identity-providers",
+        "description" : "${role_view-identity-providers}",
+        "composite" : false,
+        "clientRole" : true,
+        "containerId" : "9bac98c6-31db-4742-b3ae-1247cfca5d9f",
+        "attributes" : { }
+      }, {
+        "id" : "ffbad4be-1be5-4847-adc5-b53282653b89",
+        "name" : "view-users",
+        "description" : "${role_view-users}",
+        "composite" : true,
+        "composites" : {
+          "client" : {
+            "realm-management" : [ "query-users", "query-groups" ]
+          }
+        },
+        "clientRole" : true,
+        "containerId" : "9bac98c6-31db-4742-b3ae-1247cfca5d9f",
+        "attributes" : { }
+      } ],
+      "security-admin-console" : [ ],
+      "apiman-gateway-api" : [ ],
+      "admin-cli" : [ ],
+      "broker" : [ {
+        "id" : "ed8377db-43ac-4bf7-b2a3-9330b2fb926a",
+        "name" : "read-token",
+        "description" : "${role_read-token}",
+        "composite" : false,
+        "clientRole" : true,
+        "containerId" : "0eb83a1a-5156-4094-a286-de83130f1c8e",
+        "attributes" : { }
+      } ],
+      "kibana" : [ ],
+      "account" : [ {
+        "id" : "32668d3d-3b6d-4bfc-9107-71e52b1d3f75",
+        "name" : "view-profile",
+        "description" : "${role_view-profile}",
+        "composite" : false,
+        "clientRole" : true,
+        "containerId" : "a04ff78f-d4c7-41ce-afa7-bc28bca2ba58",
+        "attributes" : { }
+      }, {
+        "id" : "7b2bcfbd-cd5c-487d-963d-1afc11b71823",
+        "name" : "manage-account-links",
+        "description" : "${role_manage-account-links}",
+        "composite" : false,
+        "clientRole" : true,
+        "containerId" : "a04ff78f-d4c7-41ce-afa7-bc28bca2ba58",
+        "attributes" : { }
+      }, {
+        "id" : "f306a0a9-48ed-46f0-b141-52f137f140de",
+        "name" : "manage-account",
+        "description" : "${role_manage-account}",
+        "composite" : true,
+        "composites" : {
+          "client" : {
+            "account" : [ "manage-account-links" ]
+          }
+        },
+        "clientRole" : true,
+        "containerId" : "a04ff78f-d4c7-41ce-afa7-bc28bca2ba58",
+        "attributes" : { }
+      } ]
+    }
+  },
+  "groups" : [ {
+    "id" : "f0b863a6-5fed-4ced-b733-b14ed04e35dc",
+    "name" : "API-Mgmt-Administrators",
```

Comment:
I think we renamed this group to "API-Management" can you check if this is the latest version of that file, please.